.travis.yml

```diff
@@ -96,6 +96,7 @@ jobs:
         - docker-compose rm -f
 
     - stage: Push Docker image(s)
+      if: fork = false
```

Comment:
@sergei-maertens I moved the conditional here, not sure if it works, but if it is declared before the jobs, it will push the docker image before running all the tests (which is not desirable I assume?)

Reply:
I have an external PR I can rebase on this branch, we'll see in a moment :-)

Reply:
seems to work as intended, can you approve this PR so I can merge it?